public/javascripts/table_anim.js

```diff
@@ -2623,16 +2623,44 @@ anlixDocumentReady.add(function() {
                 ) +
               '</div>'+
             '</div>';
+            let wifi5StatusColor = '';
+            if (device.wifi_status_5ghz === 'Up') {
+              wifi5StatusColor = 'green-text';
+            } else {
+              wifi5StatusColor = 'red-text';
+            }
+            let wifi5Status =
+            '<div class="fas fa-circle fa-md '+wifi5StatusColor+' ml-2"'+
+            'title="'+device.wifi_status_5ghz+'"'+
+            'data-toggle="tooltip">' +
+            '</div>';
+            if (!device.wifi_status_5ghz) {
+              wifi5Status = '';
+            }
             wifi5PaneNav =
             '<a class="btn-primary text-white nav-link" '+
               'id="wifi5-pane-tab-'+index+
               '" data-toggle="tab" href="#wifi5-pane-'+index+
               '" role="tab" aria-controls="#wifi5-pane-'+index+
               '" aria-selected="false">'+
-                '5.0GHz'+
+              '5.0GHz'+
+              (device.permissions.grantWifiStatus ? wifi5Status : '' )+
```

Comment:
Acredito que aqui tenha que adicionar mais um nível de tabulação por estar dentro do `<a>`
```suggestion
                '5.0GHz'+
                (device.permissions.grantWifiStatus ? wifi5Status : '' )+
```